src/handlers/RF24StreamLogHandler.h

```diff
@@ -76,14 +71,28 @@ class RF24StreamLogHandler : public RF24AbstractLogHandler
      * @param format The format of the message
      * @param args The sequence of args
      */
-    void appendFormattedMessage(const __FlashStringHelper *format, va_list *args);
+    void appendFormat(const char format, va_list *args);
+
+#if defined (ARDUINO_ARCH_AVR)
+    void write(uint8_t logLevel,
+               const __FlashStringHelper *vendorId,
+               const __FlashStringHelper *message,
+               va_list *args);
+
+    /**
+     * @brief output a origin of the message
+     * @param vendorId The origin name.
+     */
+    void appendVendorId(const __FlashStringHelper *vendorId);
 
     /**
      * @brief output a message with the specifiers replaced with values from the sequence of @p args
      * @param format The format of the message
      * @param args The sequence of args
      */
-    void appendFormat(const char format, va_list *args);
+    void appendFormattedMessage(const __FlashStringHelper *format, va_list *args);

```

Comment:
These functions' definitions need to also be encapsulated with `#if defined (ARDUINO_ARCH_AVR) ... #endif`. [Arduino CI is showing errors related to a function definition not matching any function declarations](https://github.com/nRF24/RF24Log/pull/11/checks?check_run_id=2388123947#step:3:14) because the definitions aren't excluded on non-AVR boards the same way the declarations are in the class header.

Reply:
Oops, sorry, my bad. I will fix.